plankton/adapters/__init__.py

```diff
@@ -108,7 +113,7 @@ def import_adapter(device_name, protocol_name, device_package='devices'):
 
 
 class ForwardProperty(object):
-    def __init__(self, target_member, property_name):
+    def __init__(self, target_member, property_name, instance=None):
```

Comment:
The bright editor theme strikes again, thanks :) That should also clear up the comment below.